proto/interchain_security/ccv/consumer/v1/query.proto

```diff
@@ -52,3 +56,17 @@ message QueryParamsResponse {
   // params holds all the parameters of this module.
   Params params = 1 [ (gogoproto.nullable) = false ];
 }
+
+message QueryProviderInfoRequest {}
```

Comment:
nit: I was wondering if in general we have any naming conventions on those? E.g., the query is named `QueryProviderChainInfo` but the request and response are just `...ProviderInfo...` and _not_ `ProviderChainInfo`.

Reply:
good catch. i  changed `QueryProviderChainInfo` -> `QueryProviderInfo`